### DISCUSSION TITLE
Lesson 9: I can not test my contract on sepolia with deployerKey

### DISCUSSION THREAD

**Discussion by ely67** [Cyfrin/foundry-full-course-cu#1436] [Q&A]
When attempting to test my contract using the following command, I encountered an error:

Command: `forge test --fork-url $SEPOLIA_RPC_URL`

```plaintext
Error:
Failing tests:
Encountered 1 failing test in test/unit/RaffleTest.t.sol:RaffleTest
[FAIL. Reason: Setup failed: failed parsing $PRIVATE_KEY as type `uint256`: missing hex prefix ("0x") for hex string] setUp() (gas: 0)
```

**Comment by EngrPips** [comment]
@ely67  It says the value assigned to your $PRIVATE_KEY variable misses the "0x" starting string, i would say add the "0x" to the value of your $PRIVATE_KEY variable and then retry again.
